File: backend/GestionSetlistApp/Models/Morceau.cs
namespace GestionSetlistApp.Models
{
    public class Morceau
    {
        public int MorceauId { get; set; }
        public required string Titre { get; set; }
        public required string Artiste { get; set; }
        public string? Album { get; set; }
        public string? LienYoutube { get; set; }
        public int DureeMorceau { get; set; }

        public ICollection<MorceauSetlist> MorceauSetlists { get; set; } = new List<MorceauSetlist>();
    }
}

File: frontend/gestion-setlist/src/app/models/morceau.ts
import { MorceauSetlist } from "./morceauSetlist";

export interface Morceau {
    morceauId: number;
    titre: string;
    artiste: string;
    album: string;
    dureeMorceau: number;
    lienYoutube: string;
    morceauSetlists: MorceauSetlist[];
}

